includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -48,10 +48,24 @@ public function __construct( WC_Payments_Account $account, $gateway ) {
 	 */
 	public function init(): string {
 		global $product;
-		$price           = $product->get_price();
-		$currency_code   = get_woocommerce_currency();
-		$store_country   = WC()->countries->get_base_country();
-		$billing_country = WC()->customer->get_billing_country();
+
+		$currency_code      = get_woocommerce_currency();
+		$store_country      = WC()->countries->get_base_country();
+		$billing_country    = WC()->customer->get_billing_country();
+		$base_product_price = WC_Payments_Utils::prepare_amount( $product->get_price(), $currency_code );
+
+		$product_prices = [ 'base_product' => $base_product_price ];
+
+		foreach ( $product->get_children() as $variation_id ) {
+			$variation = wc_get_product( $variation_id );
+			if ( $variation ) {
+				$product_prices[ $variation_id ] = WC_Payments_Utils::prepare_amount( $variation->get_price(), get_woocommerce_currency() );
+			}
+		}
+
+		if ( WC()->customer ) {
```

Comment:
This if statement seems to be a merge-artifact as well.

Reply:
Ouch. thanks for catching this. I took a careful look at the other merged PR that was conflicting and removed the outdated code.